x-pack/plugins/reporting/server/browsers/chromium/paths.ts

```diff
@@ -61,16 +52,40 @@ export class ChromiumArchivePaths {
       binaryChecksum: '13baccf2e5c8385cb9d9588db6a9e2c2',
       binaryRelativePath: 'headless_shell-linux_arm64/headless_shell',
       location: 'custom',
+      revision: 856583, // FIXME: this is an older revision that is re-used from Puppeteer 8.0.0
     },
     {
       platform: 'win32',
       architecture: 'x64',
       archiveFilename: 'chrome-win.zip',
-      archiveChecksum: '64999a384bfb6c96c50c4cb6810dbc05',
-      binaryChecksum: '13b8bbb4a12f9036b8cc3b57b3a71fec',
+      archiveChecksum: '861bb8b7b8406a6934a87d3cbbce61d9',
+      binaryChecksum: 'ffa0949471e1b9a57bc8f8633fca9c7b',
       binaryRelativePath: 'chrome-win\\chrome.exe',
       location: 'common',
       archivePath: 'Win',
+      revision: 901912,
```

Comment:
Changing the `revision` is mainly what causes Kibana to download the correct version of chromium to match the version needed for Puppeteer.